tests/managers/hardware_tests.py

```diff
@@ -626,6 +626,54 @@ def test_get_hard_drive_empty(self):
 
         self.assertEqual([], result)
 
+    def test_get_hardware_guests_empty_virtualHost(self):
+        mock = self.set_mock('SoftLayer_Hardware_Server', 'getVirtualHost')
+        mock.return_value = None
+
+        result = self.hardware.get_hardware_guests(1234)
+
+        self.assertEqual(None, result)
+
+    def test_get_hardware_guests(self):
+        mock = self.set_mock('SoftLayer_Virtual_Host', 'getGuests')
+        mock.return_value = [
+            {
+                "accountId": 11111,
+                "hostname": "NSX-T Manager",
+                "id": 22222,
+                "maxCpu": 16,
+                "maxCpuUnits": "CORE",
+                "maxMemory": 49152,
+                "powerState": {
+                    "keyName": "RUNNING",
+                    "name": "Running"
+                },
+                "status": {
+                    "keyName": "ACTIVE",
+                    "name": "Active"
+                }
+            }]
+
+        result = self.hardware.get_hardware_guests(1234)
+
+        self.assertEqual([
```

Comment:
I'd prefer tests like this only test a few of the expected values, not testing if the whole object is an expected value. That way adding more data won't break the test.

`self.assertEqual("NSX-T Manager", result[0]['hostname'])`

or something like that.

Reply:
Done.